core/src/main/res/values/strings.xml

```diff
@@ -391,7 +391,7 @@
     <string name="pref_autoUpdateIntervallOrTime_every">every %1$s</string>
     <string name="pref_autoUpdateIntervallOrTime_at">at %1$s</string>
     <string name="pref_followQueue_title">Continuous Playback</string>
-    <string name="pref_pauseOnHeadsetDisconnect_title">Headphones Disconnect</string>
+    <string name="pref_pauseOnHeadsetDisconnect_title">Headphones or Bluetooth disconnect</string>
```

Comment:
This change looks like it was introduced when rebasing

Reply:
Looks like commit 5be6880989e6b8f563772893a3c1703cfd248938 sneaked in from develop. I was foolish and assumed develop had been merged into master at some point after my last rebase against develop (which was a long time ago), but I should have looked closer at the commits after rebasing against master. Sorry! I will "disappear it."